src/init/defineData/removeVariables.js

```diff
@@ -0,0 +1,23 @@
+import { set, merge } from 'd3';
```

Comment:
Looks good. This addresses #79, so I'm going to close #86 without merging and add #79 to this PR.